README.md

```diff
@@ -2,6 +2,8 @@
 
 ![Deephaven Data Labs Logo](docs/images/Deephaven_GH_Logo.svg)
 
+[Quick start for Docker user guide](https://deephaven.io/core/docs/tutorials/quickstart)
```

Comment:
We should also reference the quick start for pip.
https://deephaven.io/core/docs/tutorials/quickstart-pip/